### PR TITLE
Welcome message for students

### PR DESCRIPTION
**Pull request by brntbeer**
:wave: @githubschool/developers care to look at this change? I think it's actually a quick one that's ready to merge. Just let me know!

cc @bas @stoe

### CODE REVIEW

File: Views/Home/Index.cshtml [hunk 1]
```diff
@@ -12,5 +12,5 @@
     <p><a href="http://asp.net" class="btn btn-primary btn-large">Learn more &raquo;</a></p>
 </div>
 <div class="row">
-    <h3 title="@Model.Name" class="@helloClass">Hello @Model.Name!</h3>
+    <h3 title="@Model.Name" class="@helloClass">Hello Students!</h3>
```
Comment (stoe) [inline Views/Home/Index.cshtml]: @brntbeer, you want to remove the `title="@Model.Name"` attribute, too.

Comment (brntbeer) [inline Views/Home/Index.cshtml]: 👍 @stoe making that change now!

**Comment by brntbeer** [comment]
@stoe made those changes. take one last review?

**Comment by stoe** [comment]
Looks ✨ 
:shipit:

**Pull request merged**